### PR TITLE
Workaround for ctrl+"+" on some non-english keyboards.

### PR DESCRIPTION
**Pull request by danielfenner**
Bug: http://b/237773876

### CODE REVIEW

File: src/OrbitGl/CaptureWindow.cpp [hunk 1]
```diff
@@ -424,7 +424,11 @@ void CaptureWindow::KeyPressed(unsigned int key_code, bool ctrl, bool shift, boo
       time_graph_->GetTrackContainer()->IncrementVerticalScroll(
           /*ratio=*/-kScrollingRatioPerPageUpAndDown);
       break;
+    // Adding 93 == ']' here such that with e.g. German and Italian keyboards the ctrl+"+" shortcut
+    // works. As an unwanted side effect ctrl+"]" on an English keyboard also acts as vertical zoom.
+    // See b/237773876.
     case '+':
+    case 93:
```
Comment (karupayun) [inline src/OrbitGl/CaptureWindow.cpp:431]: Can I ask you to also add '=' as discussed online, so Ctrl + '+' works without needed to press shift? 

Context: https://doc.qt.io/qt-5/qkeysequence.html#keyboard-layout-issues

**Review by karupayun — APPROVED** [review]
Thanks for this PR! I was thinking on doing exactly this hack, so LGTM. I liked it and I think it's the best thing we can make for now.

I left a comment about adding zooming-in using '='. Can I ask you to also add "_" as another option to zoom-out (similar to '-')?

File: src/OrbitGl/CaptureWindow.cpp (hunk 1 above)
Comment (danielfenner) [reply src/OrbitGl/CaptureWindow.cpp]: Sg, added "=", "_" as discussed.

**Comment by karupayun** [comment]
Thank you Daniel!

**Pull request merged**